apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,248 @@
+package gov.cms.bfd.server.war.commons;
+
+/** Open API content short and value descriptions for each BFD API endpoint. */
+public final class OpenAPIContentProvider {
+  // NOTE: As much as we'd like to use Java 17 features for string concatentation,
+  // our code formatter completely mangles MUCH of the following concatenated text
+  // strings. So we'll fal back on tried-and-true structure to avoid
+  // unpleasantness.
+
+  /** Open API content short description for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_SHORT =
+      "The patient resource identifier (_id) to search for";
+  /** Open API content value for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_VALUE =
+      "Fetch <i>Patient<i/> data using a FHIR <i>IdType<i/> identifier; an IdType"
+          + " represents the logical identity for a resource, or as much of that"
+          + " identity that is known. In FHIR, every resource must have a <i>logical ID<i/> which is"
+          + " defined by the FHIR specification as:"
+          + "<p><code>"
+          + "Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),"
+          + " '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID"
+          + " or any other identifier pattern that meets these constraints.)</code></p><p>"
+          + "This class contains that logical ID, and can optionally also contain a relative or absolute URL"
+          + " representing the resource identity; the following are all valid values for IdType, and all might"
+          + " represent the same resource:</p><ul>"
+          + "<li><code>123</code> (just a resource\u0027s ID)</li>"
+          + "<li><code>Patient/123</code> (a relative identity)</li>"
+          + "<li><code>http://example.com/Patient/123 (an absolute identity)</code></li>";
```

Comment:
These would be more readable as text blocks and markdown.  I think @mjburling determined this is possible and google format safe.

Reply:
Yes markdown changes will be pushed later today